lambda/main_test.go

```diff
@@ -38,3 +38,13 @@ func TestParseVersion(t *testing.T) {
 	assert.Nil(t, err)
 	assert.Equal(t, "v2.6.25", res)
 }
+
+func TestIsPreRelease(t *testing.T) {
+	res1, err := isPreRelease("v0.25.29-pre%2B8224954")
```

Comment:
is the `%` a common thing you see in prerelease version strings?

Reply:
It's because the version tag comes in as url encoded.  The string we're testing is taken directly from the CloudWatch logs.